wiki/Ranking_Criteria/Ranking_Criteria_Council/en.md

```diff
@@ -12,151 +12,175 @@ Members of the Ranking Criteria Council were hand-picked to represent different
+- ![][flag_TW] [qoot8123](https://osu.ppy.sh/users/766371) (Asian mappers)
+- ![][flag_KR] [bbj0920](https://osu.ppy.sh/users/87546) (Asian players)
+- ![][flag_US] [MMzz](https://osu.ppy.sh/users/128993) (QAT)
+- ![][flag_MX] [DakeDekaane](https://osu.ppy.sh/users/1425253) (QAT)
 
 ### osu!catch members
 
-- [ZiRox](https://osu.ppy.sh/users/200768) (American mappers)
-- [Zak](https://osu.ppy.sh/users/1375955) (American players)
-- [JBHyperion](https://osu.ppy.sh/users/4879508) (European mappers)
+- ![][flag_CL] [ZiRoX](https://osu.ppy.sh/users/200768) (American mappers)
+- ![][flag_US] [Zak](https://osu.ppy.sh/users/1375955) (American players)
+- ![][flag_GB] [JBHyperion](https://osu.ppy.sh/users/4879508) (European mappers)
 - *(vacant)* (European players)
-- [Spectator](https://osu.ppy.sh/users/702598) (Asian mappers)
-- [He Ang Erika](https://osu.ppy.sh/users/2451381) (Asian players)
-- [Deif](https://osu.ppy.sh/users/318565) (QAT)
-- [Kurokami](https://osu.ppy.sh/users/260933) (QAT)
+- ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598) (Asian mappers)
+- ![][flag_SG] [He Ang Erika](https://osu.ppy.sh/users/2451381) (Asian players)
+- ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565) (QAT)
+- ![][flag_HU] [Kurokami](https://osu.ppy.sh/users/260933) (QAT)
 
 ### osu!mania members
 
-- [Fullerene-](https://osu.ppy.sh/users/2531335) (American mappers)
-- [Halogen-](https://osu.ppy.sh/users/169992) (American players)
-- [-Kamikaze-](https://osu.ppy.sh/users/2124783) (European mappers)
-- [Hayabusa](https://osu.ppy.sh/users/3104108) (European players)
-- [iJinjin](https://osu.ppy.sh/users/2141612) (Asian mappers)
-- [Cryolien](https://osu.ppy.sh/users/1626983) (Asian players)
-- [Starry-](https://osu.ppy.sh/users/2166199) (QAT)
-- [Blocko](https://osu.ppy.sh/users/4075092) (QAT)
+- ![][flag_CA] [Fullerene-](https://osu.ppy.sh/users/2531335) (American mappers)
+- ![][flag_US] [Halogen-](https://osu.ppy.sh/users/169992) (American players)
+- ![][flag_PL] [-Kamikaze-](https://osu.ppy.sh/users/2124783) (European mappers)
+- ![][flag_GB] [Hayabusa](https://osu.ppy.sh/users/3104108) (European players)
+- ![][flag_US] (![][flag_KR]) [Jinjin](https://osu.ppy.sh/users/3360737) (Asian mappers)
+- ![][flag_MY] [Cryolien](https://osu.ppy.sh/users/1626983) (Asian players)
+- ![][flag_GB] [Starry-](https://osu.ppy.sh/users/2166199) (QAT)
+- ![][flag_US] [Blocko](https://osu.ppy.sh/users/4075092) (QAT)
 
-The Ranking Criteria Council was ultimately unsuccessful. Only members involved in osu!catch promoted new proposals, while the other game modes did not make progress until the Ranking Criteira Council was disbanded and replaced with the United Beat-knights of Ranking Criteria.
+The Ranking Criteria Council was ultimately unsuccessful. Only members involved in osu!catch promoted new proposals, while the other game modes did not make progress until the Ranking Criteira Council was disbanded and replaced with the United Beat-Knights of Ranking Criteria.
```

Comment:
```suggestion
The Ranking Criteria Council was ultimately unsuccessful. Only members involved in osu!catch promoted new proposals, while the other game modes did not make progress until the Ranking Criteria Council was disbanded and replaced with the United Beat-Knights of Ranking Criteria.
```

Reply:
I think this one need to be reopened @Nivalyx . There is typo on the second sentence where `Criteira` should be `Criteria`

Reply:
oh i did not notice this earlier whoops

fixed!